docs/design-proposals/2023-04-15-ebs-br-support-multiple-k8s.md

```diff
@@ -0,0 +1,221 @@
+
+```yaml
+apiVersion: pingcap.com/v1alpha1
+kind: VolumeBackupSchedule
+metadata:
+  name: bks-federation
+  namespace: default
+spec:
+  schedule: 0 0 1 * * ?
+  pause: false
+  maxReservedTime: 7d
+  backupTemplate: {{spec of VolumeBackup}}
+```
+
+<h3>Restore</h3>
+
+In the BR Federation Manager, a new CRD, `VolumeRestore`, will be created. The user needs to specify the kubeconfig, tc, and backup storage path information for each Kubernetes cluster, and it is necessary to ensure that Kubernetes and backups are in the same AZ.
+
+```yaml
+apiVersion: federation.pingcap.com/v1alpha1
+kind: VolumeRestore
+metadata:
+  name: rt-federation
+  namespace: default
+spec:
+  clusterList:
+  - k8sClusterName: cluster-1
+    tcName: tc-1
+    tcNamespace: ns1
+    azName: az4
+    backup:
+      s3:
+        provider: aws
+        region: us-west-2
+        bucket: bucket-name
+        prefix: tc-1-bk-federation-az1
+  - k8sClusterName: cluster-2
+    tcName: tc-2
+    tcNamespace: ns2
+    azName: az5
+    backup:
+      s3:
+        provider: aws
+        region: us-west-2
+        bucket: bucket-name
+        prefix: tc-2-bk-federation-az2
+  - k8sClusterName: cluster-3
+    tcName: tc-3
+    tcNamespace: ns3
+    azName: az6
+    backup:
+      s3:
+        provider: aws
+        region: us-west-2
+        bucket: bucket-name
+        prefix: tc-3-bk-federation-az3
+  template:
+    toolImage: br-image
+    serviceAccount: sa1
+```
+
+<h4>Restore Process</h4>
+
+The restore process was originally divided into two phases, where the first phase involved restoring the TiKV volumes and the second phase involved restoring TiKV data to the resolved ts state. The first phase needs to be executed in each Kubernetes cluster, while the second phase only needs to be executed in a single Kubernetes cluster. However, there is a follow-up task after the second phase, which involves restarting TiKV pods and setting TC recoveryMode to false. This task needs to be executed in each Kubernetes cluster, so we divide the restore process into three phases.
+
+![new-restore-process-in-data-plane.png](new-restore-process-in-data-plane.png)
+
+**Before restoration:**
+
+1. Users create `TiDBCluster` CR in each data plane Kubernetes cluster and set the `recoveryMode` of all `TiDBCluster` CR to `true`, and also ensure that the TiKV replicas in each AZ are consistent with the backup.
+
+**First phase:**
+
+2. The restore controller of the control plane creates `Restore` CRs in each Kubernetes cluster with `federalVolumeRestorePhase: restore-volume`.
+3. The restore controller of the data plane restores the volumes from EBS snapshots, correctly assigns each volume to each TiKV, and starts TiKV.
```

Comment:
Please also describe the warmup step here in words and in restore process picture. @WangLe1321

Reply:
done